plugins/repogpgcheck/repogpgcheck.c

```diff
@@ -21,13 +21,13 @@ _TDNFVerifyResult(
 
     /* pContext release will free pResult. do not free otherwise */
     pResult = gpgme_op_verify_result(pContext);
-    if (!pResult)
+    if (!pResult || !(pSig = pResult->signatures))
```

Comment:
I think it would look cleaner to use `if (!pResult || !pResult->signatures)` and then leave the `for` loop unchanged.

Reply:
Wanted to save a read :) fixed it